west.yml

```diff
@@ -21,6 +21,8 @@ manifest:
   remotes:
     - name: upstream
       url-base: https://github.com/zephyrproject-rtos
+    - name: antmicro
+      url-base: https://github.com/antmicro
```

Comment:
This is not allowed in the upstream manifest.  I can create the HAL repo for you instead

Reply:
That's what I assumed, but decided I'd create a PR first and change it later. A `hal_quicklogic` repo would be very useful indeed